tests/Microsoft.Bot.Builder.TestBot/Microsoft.Bot.Builder.TestBot.csproj

```diff
@@ -1,21 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <IsPackable>false</IsPackable>
-    <GenerateDocumentationFile>false</GenerateDocumentationFile>
+    <UserSecretsId>f277272c-9113-411f-addd-a4acfa9701a2</UserSecretsId>
```

Comment:
What's this? Seems scary and should be deleted. 

... if it's not scary and needs to be kept, it needs a comment explaining why it's not scary.

Reply:
came from the asp project system... i can find out more